server/igc/IgcAnalyzer.js

```diff
@@ -143,6 +146,20 @@ function igcIsManipulated(igc) {
   return manipulated;
 }
 
+/**
```

Comment:
Und was passiert mit Leuten die wirklich ihr Vario vergessen haben zu starten?
Ich habe letzten Samstag auch mein XCTrack erst in der Luft gestartet. Okay das war sogar noch deutlich unter 900m aber trotzdem...
Evtl. sollten wir den clientseitigen Fehlertext noch ändern:
"Es scheint als würde diese igc Datei mitten im Flug beginnen. Bitte wende dich direkt an ${ADMIN_EMAIL} zur Klärung des Sachverhaltes."

Reply:
Na wie willst du denn unterscheiden zwischen Leuten, die ihr Vario absichtlich und unabsichtlich im Flug anmachen? Im Test steht doch schon, dass man sich an einen Admin wenden soll.

Reply:
Soll ich da jetzt noch weiter dran basteln, oder wollen wir das Feature nicht?

Reply:
Habe ja jetzt das mit der Elevation umgesetzt. 
Hätte das jetzt so umgebaut, dass wir gegen einen festen Wert (+200m?) zur Startplatz Elevation prüfen

Reply:
Hatte ich gesehen, aber du hattest ja auch angemerkt, dass du auch schon mal das Vario vergisst. Aber ja, 200m klingt doch gut, dann mache ich mich da mal dran.